spec/features/idv/in_person_spec.rb

```diff
@@ -10,6 +10,130 @@
     allow(IdentityConfig.store).to receive(:in_person_proofing_enabled).and_return(true)
   end
 
+  context 'ThreatMetrix review pending' do
+    let(:user) { user_with_2fa }
+
+    before do
+      allow(IdentityConfig.store).to receive(:lexisnexis_threatmetrix_enabled).and_return(true)
+      allow(IdentityConfig.store).to receive(:lexisnexis_threatmetrix_required_to_verify).
+        and_return(true)
+      allow(IdentityConfig.store).to receive(:proofing_device_profiling_decisioning_enabled).
+        and_return(true)
+    end
+
+    it 'allows the user to continue down the happy path', allow_browser_log: true do
+      sign_in_and_2fa_user(user)
+      begin_in_person_proofing(user)
+
+      # location page
+      bethesda_location = page.find_all('.location-collection-item')[1]
+      bethesda_location.click_button(t('in_person_proofing.body.location.location_button'))
+
+      # prepare page
+      complete_prepare_step(user)
+
+      # state ID page
+      complete_state_id_step(user)
+
+      # address page
+      complete_address_step(user)
+
+      # ssn page
+      select 'Reject', from: :mock_profiling_result
+      complete_ssn_step(user)
+
+      # verify page
+      expect_in_person_step_indicator_current_step(t('step_indicator.flows.idv.verify_info'))
+      expect(page).to have_content(t('headings.verify'))
+      expect(page).to have_text(InPersonHelper::GOOD_FIRST_NAME)
+      expect(page).to have_text(InPersonHelper::GOOD_LAST_NAME)
+      expect(page).to have_text(InPersonHelper::GOOD_DOB)
+      expect(page).to have_text(InPersonHelper::GOOD_STATE_ID_NUMBER)
+      expect(page).to have_text(InPersonHelper::GOOD_ADDRESS1)
+      expect(page).to have_text(InPersonHelper::GOOD_CITY)
+      expect(page).to have_text(InPersonHelper::GOOD_ZIPCODE)
+      expect(page).to have_text(Idp::Constants::MOCK_IDV_APPLICANT[:state])
+      expect(page).to have_text('9**-**-***4')
+
+      # click update state ID button
+      click_button t('idv.buttons.change_state_id_label')
+      expect(page).to have_content(t('in_person_proofing.headings.update_state_id'))
+      click_button t('forms.buttons.submit.update')
+      expect(page).to have_content(t('headings.verify'))
+
+      # click update address button
+      click_button t('idv.buttons.change_address_label')
+      expect(page).to have_content(t('in_person_proofing.headings.update_address'))
+      click_button t('forms.buttons.submit.update')
+      expect(page).to have_content(t('headings.verify'))
+
+      # click update ssn button
+      click_button t('idv.buttons.change_ssn_label')
+      expect(page).to have_content(t('doc_auth.headings.ssn_update'))
+      fill_out_ssn_form_ok
+      click_button t('forms.buttons.submit.update')
+      expect(page).to have_content(t('headings.verify'))
```

Comment:
I think it'd be safe to say this is already covered in the spec above? If this one is primarily concerned with ThreatMetrix.

```suggestion
```

Reply:
I think these specific assertions were mischaracterized as "happy path" (though considering how slow the tests run, I'm fine to have made the practical trade-off). These are exercising scenarios where a user would want to go back and change values they'd entered on previous screens, which is optional. It's similar to clicking "Change" to change one's address in the unsupervised flow. Since we're covering it in the previous test, and since it doesn't seem to be critically relevant to ThreatMetrix, it could marginally improve the runtime of the tests to avoid checking these things again.